lxd/daemon.go

```diff
@@ -573,13 +573,13 @@ func (d *Daemon) Init() error {
 
 	/* Print welcome message */
 	if d.MockMode {
-		shared.LogInfo("LXD is starting in mock mode",
+		shared.LogDebug("LXD is starting in mock mode",
```

Comment:
So I converted some back from `LogDebug*()` to `LogInfo*()` but e.g. kept the AppArmor related ones at `LogWarn*()`.